packages/twilio-worker-auth/src/twilioWorkerAuthMiddleware.ts

```diff
@@ -40,7 +40,7 @@ declare global {
  * IMPORTANT: This kind of static key acces should never be used to retrieve sensitive information.
  */
 const canAccessResourceWithStaticKey = (path: string, method: string): boolean => {
-  return path === '/postSurveys' && method === 'POST';
+  return path.endsWith('/postSurveys') && method === 'POST';
```

Comment:
Could we accidentally expose more endpoints if we ever introduce regex/catchall expressions? Something like `/privatePath/postSurveys` could en up giving access to resources under `/privatePath`? I don't think so, but is worth asking.